implementations/rust/ockam/ockam_core/src/vault/types.rs

```diff
@@ -70,33 +70,33 @@ cfg_if! {
     }
 }
 
-/// Binary representation of a Secret.
+/// Binary representation of a Private key.
 #[derive(Serialize, Deserialize, Clone, Zeroize, Encode, Decode)]
 #[zeroize(drop)]
 #[cbor(transparent)]
-pub struct SecretKey(#[n(0)] SecretKeyVec);
+pub struct PrivateKey(#[n(0)] PrivateKeyVec);
```

Comment:
Could you please elaborate what is the reasoning on that renaming?

Reply:
Oh, I thought renaming SecretVault -> KeyVault and SecretKey -> PrivateKey are two separate things, are they related?